e2e/Enclave.mk

```diff
@@ -130,6 +129,6 @@ $(Signed_RustEnclave_Name): $(RustEnclave_Name)
 
 .PHONY: enclave
 enclave:
-	@cd enclave && RUSTFLAGS=$(RUSTFLAGS) cargo build $(CARGO_TARGET) $(CARGO_FEATURES)
+	cd enclave && RUSTFLAGS=$(RUSTFLAGS) cargo build $(CARGO_TARGET) $(CARGO_FEATURES)
```

Comment:
Is this change needed?

Reply:
Sorry, it's my mistake. I'll revert it. 🙇🏼‍♂️